### ISSUE TITLE
Can not compile AospExtended - Huawei Berlin

### ISSUE THREAD

**Issue by pomelohan** [AospExtended/manifest#22]
I am from China, so I used Google Translate.
I have completed the source code synchronization, but the implementation of the preparation of the environment when the order suggested that the following: 
root@ubuntu: /media/psf/Android #. build/ envsetup.sh
including device / honor / berlin / vendorsetup.sh
including vendor / aosp / vendorsetup.sh
including sdk / bash_completion / adb.bash
root @ ubuntu: / media / psf / Android # lunch lineage_berlin-userdebug
build / core / product_config.mk: 234: *** Can not locate config makefile for product "lineage_berlin".
Device berlin not found. Attempting to retrieve device repository from AospExtended-Devices Github (http://github.com/AospExtended-Devices).
Repository for berlin not found in the AospExtended-Devices Github repository list.
If this is in error, you may need to manually add it to your .repo / local_manifests / aex_manifest.xml
build / core / product_config.mk: 234: *** Can not locate config makefile for product "lineage_berlin".

** Do not have a product spec for: 'lineage_berlin'
** Do you have the right repo manifest?
build / core / product_config.mk: 234: *** Can not locate config makefile for product "lineage_berlin".
build / core / product_config.mk: 234: *** Can not locate config makefile for product "lineage_berlin".

** Do not have a product spec for: 'lineage_berlin'
** Do you have the right repo manifest?

Hope to answer my issues!
Thank you very much!
Hava a good day!

source page:
https://github.com/OpenKirin/android_vendor_honor_berlin/tree/7.x
https://github.com/OpenKirin/android_kernel_honor_berlin/tree/7.x
https://github.com/OpenKirin/android_device_honor_berlin/tree/7.x

Android 7.1.2

**Comment by ghost** [comment]
The lunch combo is wrong. You are using "add_lunch_combo lineage_berlin-userdebug", replace lineage with aosp. You can also take a hint from this commit https://github.com/theimpulson/android_device_asus_Z00L/commit/d8351321dd2f513686ffd7341fc60539327fafd8

**Comment by ishubhamsingh** [comment]
you need to edit your trees for aex and you need to use aosp_berlin as lunch target

**Comment by pomelohan** [comment]
Thank you very much . Let me try again :)

**Comment by pomelohan** [comment]
root@ubuntu:/media/psf/Android/aex# . build/envsetup.sh
including device/honor/berlin/vendorsetup.sh
including vendor/aosp/vendorsetup.sh
including sdk/bash_completion/adb.bash
root@ubuntu:/media/psf/Android/aex# lunch aosp_berlin-userdebug
build/core/product_config.mk:234: *** Can not locate config makefile for product "aosp_berlin".  Stop.
Device berlin not found. Attempting to retrieve device repository from AospExtended-Devices Github (http://github.com/AospExtended-Devices).
Repository for berlin not found in the AospExtended-Devices Github repository list.
If this is in error, you may need to manually add it to your .repo/local_manifests/aex_manifest.xml
build/core/product_config.mk:234: *** Can not locate config makefile for product "aosp_berlin".  Stop.

** Don't have a product spec for: 'aosp_berlin'
** Do you have the right repo manifest?
build/core/product_config.mk:234: *** Can not locate config makefile for product "aosp_berlin".  Stop.
build/core/product_config.mk:234: *** Can not locate config makefile for product "aosp_berlin".  Stop.

** Don't have a product spec for: 'aosp_berlin'
** Do you have the right repo manifest?

**Comment by ghost** [comment]
Did you edited the tree like the commit I gave to you?

**Comment by pomelohan** [comment]
I tried to modify it, and still prompted this. Can I go to Fork's branch and Commit?Please Help me!! Thank you :(

**Comment by ghost** [comment]
Commit changes you did, in a repo and give link to me to see. In case you need help on commiting and uploading ask.

**Comment by pomelohan** [comment]
https://github.com/RainSnowZ/android_device_honor_berlin/commit/605837c9579a996b477be7e7180c2d32a4d674f6
Help Me please!Thank you

**Comment by ghost** [comment]
Try this https://github.com/theimpulson/android_device_honor_berlin/tree/Nougat If doesn't works, ping me on telegram. My username is TheImpulson. Close this issue BTW. It's not ROM concerned.
